flagsmith/flagsmith.py

```diff
@@ -364,9 +372,7 @@ def _get_json_response(
         self,
         url: str,
         method: str,
-        body: typing.Optional[
-            typing.Union[Identity, typing.Dict[str, JsonType]]
-        ] = None,
+        body: typing.Optional[JsonType] = None,
     ) -> typing.Any:
```

Comment:
Shouldn't the return type be `JsonType`?

Reply:
The return type here corresponds to `response.json()` type, which is `typing.Any`.
Casting to a specific type is responsibility of the caller, see e.g. https://github.com/Flagsmith/flagsmith-python-client/blob/b0e85d8f84c1b8c5ab6c6bee51017b59402dda9e/flagsmith/flagsmith.py#L358-L359.